ui/scss/component/nag.scss

```diff
@@ -30,6 +30,7 @@ $nag-error-z-index: 999;
   top: 0;
   bottom: auto;
   z-index: 1 !important; /* booooooo */
+  border-radius: var(--border-radius) var(--border-radius) 0 0;
 }
 
 .nag--relative {
```

Comment:
Spacing tweaks

![image](https://user-images.githubusercontent.com/64950861/148081563-31a64a6e-f10d-48fb-9d1f-f46857f3e320.png)

![image](https://user-images.githubusercontent.com/64950861/148081760-b15e3d10-0eb8-40b2-b783-176f41f1df79.png)

Reply:
Fixed the first one. Added border radius & padding. However, I'm unable to reproduce the 2nd one. I've tested this on multiple devices on multiple pages. How does this happen? I've tested it in the comment section for files, in the replies for comments, in livechats and in the replies in the notifications.
![image](https://user-images.githubusercontent.com/94839278/148099043-0bd1ef23-fcca-4827-b281-528df7471e0f.png)

Reply:
Hm ... it's in comments section for files.  Click either LBC or $
![image](https://user-images.githubusercontent.com/64950861/148103737-c6d19067-fc34-4b63-afa4-18029bacc451.png)

Reply:
My fault :man_facepalming: The issue is fixed and I've also optimized some other elements in the comment editor.